amman/src/validator/solana-validator.ts

```diff
@@ -61,6 +63,20 @@ export async function buildSolanaValidatorArgs(
     }
   }
 
+  if (features) {
```

Comment:
```suggestion
  if (features != null) {
```
One last nit (I like to be explicit about existence checks instead of relying on bool coercion).
This checks for both `null` and `undefined`.

Reply:
Don't we want it to skip this if it is null or undefined? I mean we define that the only states we want it to work in is array of strings or a string itself. So if we would add != null it would essentially do the same thing as it is doing now but undefined would go into the inner block. If you prefer to be more explicit then I think I would rather do ```if(features !== null && features !== undefined)```. What do you think?

Reply:
Oh wait I just reread what you wrote! So != null does both? Man javascript is magical